pkg/collect/dns.go

```diff
@@ -21,7 +21,8 @@ import (
 )
 
 const (
-	dnsUtilsImage = "registry.k8s.io/e2e-test-images/jessie-dnsutils:1.3"
+	dnsUtilsImage       = "registry.k8s.io/e2e-test-images/jessie-dnsutils:1.3"
```

Comment:
[Debian jessie EOL'd in 2020](https://www.debian.org/News/2020/20200709). Why not just use something like `busybox` or `alpine`?

Reply:
Above image is used because we are following the troubleshooting steps in https://kubernetes.io/docs/tasks/administer-cluster/dns-debugging-resolution

I didn't use busybox/alpine because they have problem with inconsistent DNS resolution due to musl implementation. It looks like it has been [fixed](https://gitlab.alpinelinux.org/alpine/tsc/-/issues/43#note_306270), I will switch to `alpine v3.20`